City-of-Helsinki/helsinki-paatokset
# PP-483: Misc fixes

This PR:
- Adds support for `del` and `ins` elements in decision content
- Improves decision and motion URL fetching on meeting pages
- Adds links to motions on meeting decision announcement content
- Adds an anchor link to voting results on decisions that have voting data
- Enables sitemap generation

**To test:**
- Checkout branch, run `make drush-cim drush-cr`
- Go to https://helsinki-paatokset.docker.so/sitemap.xml , which should result in a 404
- Pull in data for testing:
```
drush mim ahjo_decisionmakers:all
drush ap:update meetings 00400202324 -v
drush ap:update cases hel-2022-005947 -v
drush ap:update cases hel-2022-000782 -v
drush ap:update decisions 34145abe-1fdd-c320-8296-7edd98800002 -v
drush ap:update decisions 20da5659-72d8-49e5-ad8b-dcb74536e20e -v
drush ap:gm -v
```
- Reload the sitemap.xml page. You should get a sitemap now
- Open this page: https://helsinki-paatokset.docker.so/fi/paattajat/kaupunginhallitus/asiakirjat/00400202324
  - The agenda list should have links to motions
  - Scroll down to the decision announcement part and open the accordion. Each should have a "Avaa esitys" link that opens the related motion. The links should open in a new tab.
- Open this page: https://helsinki-paatokset.docker.so/fi/asia/hel-2022-005947/20da5659-72d8-49e5-ad8b-dcb74536e20e
  - In the content there should be a link with this title: "Katso äänestystulos taulukkona". When clicked, it should scroll down to the Voting results section and open the accordion item automatically
- Open this page: https://helsinki-paatokset.docker.so/fi/asia/hel-2022-000782/34145abe-1fdd-c320-8296-7edd98800002
  - When you scroll down, you should see some content with the "line-through" style. If you inspect the HTML it should be marked using the `del` element.
- Read the code changes to make sure there's nothing weird
